### ISSUE TITLE
Annotated parameter is missing nullable type

### ISSUE THREAD

**Issue by vloth** [adelsz/pgtyped#112]
For the following sql
```sql
/*
   @name addTodo
   @param todo -> (description, done, duedate)
 */
INSERT INTO todo(description, done, duedate) VALUES :todo;

/* @name addTodoNoObjectParam */
INSERT INTO todo(description, done, duedate)
VALUES (:description, :done, :duedate);
```

I get different type system:
```ts
/** 'addTodo' parameters type */
export interface IAddTodoParams {
  todo: {
    description: string,
    done: boolean,
    duedate: Date
  };
}

/** 'addTodoNoObjectParam' parameters type */
export interface IAddTodoNoObjectParamParams {
  description: string | null | void;
  done: boolean | null | void;
  duedate: Date | null | void;
}
```
Is `duedate` type for `addTodo` really supposed to be just `Date`?   
(the column duedate in the database is a nullable timestamp)

**Comment by adelsz** [comment]
Hm, looks like a bug. I will look into it.

**Comment by adelsz** [comment]
Fixed in #115
